### PR TITLE
[Dropdown] preventing change trigger option explained

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
Added parameters to prevent the change trigger according to https://github.com/fomantic/Fomantic-UI/pull/1092

## Screenshots
image

image

### CODE REVIEW

File: server/documents/modules/dropdown.html.eco [hunk 1]
```diff
@@ -2889,16 +2889,16 @@ themes      : ['Default', 'GitHub', 'Material']
           <td>Hides dropdown</td>
         </tr>
         <tr>
-          <td>clear</td>
-          <td>Clears dropdown of selection</td>
+          <td>clear (preventChangeTrigger)</td>
+          <td>Clears dropdown of selection. Set <code>preventChangeTrigger</code> to <code>true</code> to omit the change event</td>
```
Comment (exoego) [inline server/documents/modules/dropdown.html.eco]: I think documenting default values for each new parameter helps users.

File: server/documents/modules/dropdown.html.eco [hunk 2]
```diff
@@ -2889,16 +2889,16 @@ themes      : ['Default', 'GitHub', 'Material']
...
         </tr>
         <tr>
           <td>hide others</td>
           <td>Hides all other dropdowns that is not current dropdown</td>
         </tr>
         <tr>
-          <td>restore defaults</td>
-          <td>Restores dropdown text and value to its value on page load</td>
+          <td>restore defaults (preventChangeTrigger)</td>
+          <td>Restores dropdown text and value to its value on page load. Set <code>preventChangeTrigger</code> to <code>true</code> to omit the change event</td>
```
Comment (exoego) [inline server/documents/modules/dropdown.html.eco]:
```suggestion
          <td>Restores dropdown text and value to its value on page load. Set <code>preventChangeTrigger</code> to <code>true</code> to omit the change event. (default: <code>false</code>)</td>
```

File: server/documents/modules/dropdown.html.eco (hunk 1 above)
Comment (lubber-de) [reply server/documents/modules/dropdown.html.eco]: @exoego Which parameters are you referring to? The only parameter is explained here 🤔

File: server/documents/modules/dropdown.html.eco (hunk 2 above)
Comment (lubber-de) [reply server/documents/modules/dropdown.html.eco]: 🤔 What should i change?

File: server/documents/modules/dropdown.html.eco (hunk 1 above)
Comment (lubber-de) [reply server/documents/modules/dropdown.html.eco]: sorry, did not recognized your suggestion... 🙄 😄

**Pull request merged**